### ISSUE TITLE
MyReadingManga skipping some [eng] manga

### ISSUE THREAD

**Issue by happywillow0** [inorichi/tachiyomi-extensions#640]
Bug: MyReadingManga skipping some manga due to how titles and links are formatted. 

Tachiyomi version dev r1458
MyReadingManga version 1.2.15 

When I was looking at the website to fix a jpeg thumbnail issue I noticed that the app was not pulling in all the manga labeled as [Eng]. 

Looking though the code, the import is for english manga is `element.select("a[rel][href*=-$lang]")` which is looking for a html link that contains "eng" which works nicely for most of the links.  

However, there was at least two on the front page as of writing which did not fit the rule. When browsing the site, I can see that the title says [Eng] but the url doesn't. Examples as follows:
`<a class="entry-title-link" rel="bookmark" href="https://myreadingmanga.info/dream-behind-blue-eyes-tatumi-akaboshi/">[Tatumi Akaboshi] The dream behind blue eyes – Attack on Titan dj [Eng]</a>`
`<a class="entry-title-link" rel="bookmark" href="https://myreadingmanga.info/collector-by-miku-sagawa-sable/">[Miku Sagawa (Sable)] Collector – Attack on Titan dj [Eng]</a>`

I don't know enough about the code yet so I don't know if it is possible or how easy a fix may be. I wanted to at least document it for a future update for who ever can figure it out. 

Thank you for your time and effort.

**Comment by happywillow0** [comment]
Thanks for taking a look. 

I'm was double checking / verifying the fix I'm looking at the home page and going 5 pages deep, and I really don't know how I found the two exceptions to the title vs url rule. Currently all the other ones have correct urls that end or contain -eng so maybe it was just another bug in their system when generating the page urls.

All in all, I don't know if I created work for nothing but thanks anyways for taking a shot at it.
